nuc_morph_analysis/analyses/neighbor_of_X/figure_mitotic_filtering_examples.py

```diff
@@ -0,0 +1,212 @@
+#%%
```

Comment:
Can we update this example to include the backwards implimentation from formation as well as forward from breakdown?? I think you had asked me about this earlier if it was okay to just do from breakdown, but now that we have written the legend and methods I know what that detail is and because the formation event is depicted in the figure it doesnt really make sense to leave that out. Hopefully its not tooo hard to add!

Reply:
done!